src/main/java/team/chisel/ctm/client/model/parsing/ModelLoaderCTM.java

```diff
@@ -15,7 +15,7 @@
 
 	INSTANCE;
 
-	private static final Map<Integer, IModelParser> parserVersions = ImmutableMap.of(1, new ModelParserV1());
+	private static final Map<Integer, IModelParser> parserVersions = Map.of(1, new ModelParserV1());
```

Comment:
This file still contains the import for Guava's ImmutableMap class

Reply:
removed